environment.yml

```
@@ -0,0 +1,7 @@
+# configuration for binder
+
+dependencies:
+- python=3.10
+- pip
+- pip:
+  - -r ./requirements.txt
```

```suggestion
- plasmapy == 0.8.1
```
The rest of the dependencies should be handled by this automagically, so we shouldn't need to do any `pip` installations.

---

@namurphy this will only install the `install_requires` of `plasmapy` and not the other stuff like `sphinx`  ... we should still just use `-r requirements.txt`.

---

Would we need `sphinx` for running a binder notebook?

---

I see what you're getting at...I guess not 😂

---

However, the binder pkg installer is not properly resolving package conflicts.  `numba` requires `numpy<=1.21`, but `numpy==1.23` is being installed.  As a result, `plasmapy` can not be imported.

---

Yep, trying to fix that in #37.  Still having problems...